src/ore/src/str.rs

```diff
@@ -314,37 +342,28 @@ impl<const MAX: usize> MaxLenString<MAX> {
     /// let smol = ShortString::new(too_long.to_string());
     /// assert!(smol.is_err());
     /// ```
-    ///
-    pub fn new(s: String) -> Result<Self, String> {
+    fn try_from(s: String) -> Result<Self, Self::Error> {
```

Comment:
Slight preference to retain the `MaxLenString::new` method and then `try_from(...)` calls `MaxLenString::new`

Reply:
I like to limit to just one way to do things, but I don't feel strongly about it at all. Adjusted per your preference!